### ISSUE TITLE
[Pagination] a11y issues with next/previous tooltips

### ISSUE THREAD

**Issue by m4thieulavoie** [Shopify/polaris-react-archive#7953]
## Issue summary

When a11y testing this component, we noticed that `Pagination` removes the next/previous tooltips from the DOM, making those buttons described by a non-existing DOM element.

## Expected behavior

Screen readers should be able to know what the next/previous buttons do in the UI

## Actual behavior

Screen reader can't announce next/previous buttons

## Steps to reproduce the problem

1. First step
1. Second step

## Reduced test case

The best way to get your bug fixed is to provide a reduced test case. This [CodeSandbox template](https://codesandbox.io/s/q82mlq0m26) is a great starting point.

## Specifications

- Are you using the React components? (Y/N):
- Polaris version number:
- Browser:
- Device:
- Operating System:

Or run `npx envinfo --system --binaries --browsers --npmPackages react,react-dom,@shopify/polaris` to provide specifications on your environment including version numbers, browser, device, and operating system.

Paste the results here:

```bash

```

**Comment by m4thieulavoie** [comment]
After digging into it, it looks like this would be a quite tricky thing to fix, but I'm also wondering whether this is a real a11y issue. After all, the screen reader _does_ read out the label. The only thing that seems to really complain is axe. I'll leave it to the team, but I think it's safe to close that issue
